src/components/CheckBoxFilter.ts

```diff
@@ -0,0 +1,39 @@
+import { ChangeEvent, Component, createElement } from "react";
+
+export interface CheckboxFilterProps {
+    isChecked: boolean;
+    handleChange: (value: boolean) => void;
+}
+
+interface CheckboxFilterState {
+    isChecked: boolean;
+}
+
+export class CheckboxFilter extends Component<CheckboxFilterProps, CheckboxFilterState> {
+    constructor(props: CheckboxFilterProps) {
```

Comment:
Add space

Reply:
Same as above. Not necessary, since there are no private member variables.